tx_salaries/utils/transformers/base.py

```diff
@@ -115,11 +117,18 @@ def generic_transform(labels, source, record_class):
     used directly via ``transform_factory``.
     """
     data = []
+    warnings = []
     for raw_record in source:
         record = record_class(dict(zip(labels, raw_record)))
         if record.is_valid:
             data.append(record.as_dict())
-    return data
+        else:
+            warnings.append("WARNING: RECORD INVALID; {0}".format(record.data))
+    if len(warnings) != 0 and record_class.REJECT_ALL_IF_INVALID_RECORD_EXISTS:
```

Comment:
Well, the PEP8 gods apparently made a commandment, so I better make it so
https://www.python.org/dev/peps/pep-0008/#programming-recommendations